core/testing/func_reverse_test.go

```diff
@@ -0,0 +1,29 @@
+package testing
+
+import "testing"
+
+func Test_Func_Reverse_NoChars(t *testing.T) {
+	rsl := `
+a = "hello"
+print(reverse(a))
+`
+	setupAndRunCode(t, rsl, "--color=never")
+	assertOnlyOutput(t, stdOutBuffer, "olleh\n")
+	assertNoErrors(t)
+	resetTestState()
+}
+
+func Test_Func_Reverse_With_Nums(t *testing.T) {
+	rsl := `
+a = 123
+print(reverse(a))
+`
+	setupAndRunCode(t, rsl, "--color=never")
+	expected := `Error at L3:15
+
+  print(reverse(a))
+                ^ Got "int" as the 1st argument of reverse(), but must be: string
+`
+	assertError(t, 1, expected)
+	resetTestState()
```

Comment:
Two more things I'd like to test:

- empty strings
- strings with escaped characters like `\t` or `\n`, wanna make sure it doesn't become e.g. `n\` or something dumb like that :)